12/umbraco-cms/content-delivery-api.md

```diff
@@ -440,19 +456,115 @@ Which properties to expand and therefore include in the output if they refer to
 
 {% swagger-response status="400: Bad Request" description="Invalid request" %}
 
+{% endswagger-response %}
+
+{% swagger-response status="404: Not Found" description="Start-Item not found" %}
+
 {% endswagger-response %}
 {% endswagger %}
 
-## Feature Capabilities
+### Query parameters
+
+The Content Delivery API provides a number of query parameters that allow you to customize the content returned by the API to fit your needs. For each endpoint, the relevant query parameters are already specified within their corresponding documentation above. In addition to standard parameters like `skip` and `take`, the API provides several possibilities for the value of `expand`, `fetch`, `filter` and `sort` parameters. Below are the options supported out-of-the-box.
+
+{% hint style="info" %}
+You can extend the built-in selectors, filters, and sorting capabilities of the Delivery API by creating your own custom selector, filter or sort query handlers.
```

Comment:
Shortened this sentence a bit, by removed the duplicate mention of "selector, filter and sorting".
Let me know if it doesn't make sense!

```suggestion
You can extend the built-in selector, filter, and sorting capabilities of the Delivery API by creating your own custom query handlers.
```